jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/CodeModernizerSessionContext.kt

```diff
@@ -209,13 +258,15 @@ data class CodeModernizerSessionContext(
             }
         } catch (e: ProcessNotCreatedException) {
             val error = "./mvnw failed to execute as its likely not a unix machine"
+            buildlogBuilder.appendLine("./mvnw failed to execute: $e")
             CodetransformTelemetry.mvnBuildFailed(
                 codeTransformSessionId = CodeTransformTelemetryState.instance.getSessionId(),
                 codeTransformMavenBuildCommand = CodeTransformMavenBuildCommand.Mvnw,
                 reason = error
             )
             LOG.warn { error }
```

Comment:
Should we log the actual error message? And also in the below catch block?

Reply:
yes, let me fix